### ISSUE TITLE
Favicon fetching incompatible with django-storages S3

### ISSUE THREAD

**Issue by samjeckert** [fabiocaccamo/django-admin-interface#128]
The generated favicon URL is broken when the static resource is stored on S3 via django-storages. The code [here](https://github.com/fabiocaccamo/django-admin-interface/blob/ca71642ecfa483498da311c4196fe75344e62325/admin_interface/templates/admin_interface/favicon.html#L6) adds `?v={version}` to the end of the URL, but django-storages already has to add parameters to authenticate with S3. The result is a final URL with two `?`'s, which does not work. 

https://<s3_bucket>.s3.amazonaws.com/admin-interface/favicon/favicon.ico?AWSAccessKeyId=<access_key_id>&Signature=<signature>&Expires=<expires>?v=0.15.6

**Comment by merwok** [comment]
Just a precision: this project is compatible with django-storages and S3, if you are using cloudfront (a public distribution) to host your images.

**Comment by C0deZLee** [comment]
Same problem here

**Comment by bastiaan85** [comment]
One option to resolve this may be to implement a custom `vstatic` tag to support a version parameter, like shown [here](https://stackoverflow.com/a/43762942). Eg quick and dirty:

```
# django < 2.1
from django.contrib.staticfiles.templatetags.staticfiles import static
# django >= 2.1
from django.templatetags.static import static

@register.simple_tag
def vstatic(path, version):
    url = static(path)
    sep = '&' if '?' in url else '?'
    return f'{url}{sep}v={version}'
```

or with proper url parsing

```
from urllib.parse import urlencode, urlparse, parse_qsl, urlunparse

# django < 2.1
from django.contrib.staticfiles.templatetags.staticfiles import static
# django >= 2.1
from django.templatetags.static import static

@register.simple_tag
def vstatic(path, version):
    url = static(path)
    parsed = urlparse(url)
    values = dict(parse_qsl(parsed.query))
    if 'v' in values or 'version' in values:    # if you mean to make the provided version optional
        return url
    parsed = parsed._replace(query=urlencode(values))
    return urlunparse(parsed)
```

then in favicon.html

`<script type="text/javascript" src="{% vstatic 'admin_interface/favico/favico-0.3.10-patched.min.js' version %}"></script>`

**Comment by fabiocaccamo** [comment]
I will just remove the version parameter, to avoid caching issues consider to use the built-in django [ManifestStaticFilesStorage](https://docs.djangoproject.com/en/3.2/ref/contrib/staticfiles/#django.contrib.staticfiles.storage.ManifestStaticFilesStorage).

**Comment by fabiocaccamo** [comment]
Fixed within `0.17.1` version.
